ktor-client/ktor-client-core/jvm/test/WebSocketTest.kt

```diff
@@ -0,0 +1,42 @@
+import io.ktor.client.features.websocket.*
+import io.ktor.http.cio.websocket.*
+import kotlinx.coroutines.channels.*
+import kotlin.coroutines.*
+import kotlin.test.*
+
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+class WebSocketTest {
+
+    @Test
+    fun testAsDefault() {
+        val feature = WebSockets(42, 16)
+        val session = object : WebSocketSession {
```

Comment:
What prevents the direct creation of `WebSocketSession`?

Reply:
It's an interface so we need connected channels to create any `real` instance. So I think it's enough to use mock for the test.